app/models/tag.rb

```diff
@@ -201,4 +201,11 @@ def self.find_research_notes(tagnames, limit = 10)
         .order('node_revisions.timestamp DESC')
         .limit(limit)
   end
+
+  def followers_who_dont_follow_tags(tags)
+    tag_followers = self.subscriptions.map {|user| user.user_id}
```

Comment:
There's a function for collecting all the properties from an array of objects, `.collect(&:id)` will return an array of all `id`s from an array of users. Could help to make it more readable!

But I guess if you're looking for followers, you can do: `self.subscriptions.collect(&:user) -- see: https://github.com/publiclab/plots2/blob/master/app/models/tag_selection.rb

Although maybe you need to refactor that to use `rusers`!